web/src/client/storage.js

```diff
@@ -320,13 +323,28 @@ class ProposalManager {
         };
       };
 
+      const buildInstallationDevices = (proxy, devices) => {
+        const findDevice = (devices, name) => {
+          const device = devices.find(d => d.name === name);
+
+          if (device === undefined) console.log("D-Bus object not found: ", name);
+
+          return device;
+        };
+
+        const names = proxy.SystemVGDevices.filter(n => n !== proxy.BootDevice).concat([proxy.BootDevice]);
+        return names.map(dev => findDevice(devices, dev));
+      };
+
       return {
         settings: {
           bootDevice: proxy.BootDevice,
           lvm: proxy.LVM,
+          spacePolicy: proxy.SpacePolicy,
           systemVGDevices: proxy.SystemVGDevices,
           encryptionPassword: proxy.EncryptionPassword,
           volumes: proxy.Volumes.map(this.buildVolume),
+          installationDevices: buildInstallationDevices(proxy, systemDevices)
```

Comment:
Installation devices is not a setting but some information we can get from the settings. I would extract it from settings, offerting a separate funtion to get the list: `client.installationDevices()`.

Reply:
Added a comment.